PanelSearch/SQL_functions.py

```diff
@@ -8,7 +8,7 @@
 from sqlalchemy import create_engine
 
 # this is my login to my local SQL server - works for local testing purposes but won't sync.
-engine = create_engine("mysql+pymysql://root:birbtime@localhost/panelsearch")
+#engine = create_engine("mysql+pymysql://sql8665490:sn7HJyzVgH@sql8.freesqldatabase.com/sql8665490")
```

Comment:
ideally we wouldn't want to hardcode SQL credentials but maybe store them separately? Or ask user for input?

Reply:
Something worth thinking about - I'll try to look at this later but for now I'll leave it as is